hazyCalculator.js

```diff
@@ -3,16 +3,17 @@ function isSkippedValue(value) {
 }
 
 function isNumericValue(value) {
-  return !isNaN(value)
+  return !isNaN(value) && value != ''
 }
 
 function isNothingValue(value) {
-  return value === null
+  return value === null || value === undefined
 }
 
 function isAcceptableValue(value) {
   const operators = ['+', '-', '*', '/']
-  return typeof value === Number || operators.includes(value)
+  const type = typeof Number(value)
+  return type === 'number' || operators.includes(value)
```

Comment:
By doing this here you now have 2 definitions of what is a number in your application. Here you say that if something can be typecast into a number it is numeric. But in `isNumericValue` you say that (in a different way with `isNaN`) as well as if it isn't an empty string. Why not leverage `isNumericValue` here instead of having a second definition of numeric?